### DISCUSSION TITLE
Viewing code become lagging in Rstudio when run chunck above

### DISCUSSION THREAD

**Discussion by zhengchencai** [quarto-dev/quarto-cli#6685] [Q&A] [answered]
### Description

Hi there,

When I run all the chunks above in RStudio, it becomes nearly impossible to scroll through the editor to view the code. This is particularly frustrating when the chunks take a long time to run, as I have to wait until the computation finishes. Is this expected behavior, or are there settings in RStudio that might address this? I don't recall encountering this issue previously with RMarkdown. Thank you for your assistance.

**Comment by mcanouil** [comment, marked as answer]
Code cell computations in your case are done in R thus via knitr.
There is not much changes in Quarto compare to RMarkdown in that matter.
Note that this repository is not about RStudio but about Quarto CLI.

For questions or reports regarding the IDE, it is better to do so in the following repository and to provide the means to reproduce what you are experiencing which could be the consequences of many factors, not only the IDE itself.

https://github.com/rstudio/rstudio
